src/Chainweb/Pact/PactService.hs

```diff
@@ -196,39 +195,42 @@ serviceRequests memPoolAccess reqQ = do
             LocalMsg LocalReq{..} -> do
                 r <- try $ execLocal _localRequest
                 case r of
-                  Left e -> liftIO $ putMVar _localResultVar $ Left e
-                  Right r' -> liftIO $ putMVar _localResultVar r'
+                  Left (SomeException e) -> liftIO $ putMVar _localResultVar $ toPactInternalError e
```

Comment:
I assume Left can only be of type SomeExecption ?

Reply:
This is an unusual one, where we don't think any exceptions will fire but "just in case". Also, the `SomeException` specializes `try` otherwise it wouldn't compile.